libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -47,7 +47,21 @@
     ToolConfig,
     VideoMetadata,
 )
+
+from google.generativeai.caching import CachedContent  # type: ignore[import]
```

Comment:
could we avoid bringing additional dependencies like `google.generativeai`, please? We should use `google.ai.generativelanguage_v1beta.types`.